packages/@core.next/apollo.jsx

```diff
@@ -49,7 +50,21 @@ let createApolloClient = (initialState, ctx) => {
             fetch: (isOnClientSide && window.fetch) ? window.fetch : fetch,
             headers: (ctx && ctx.req) ? ctx.req.headers : undefined,  // allow to use client cookies on server side requests
         }),
-        cache: new InMemoryCache().restore(initialState || {}),
+        cache: new InMemoryCache({
+            typePolicies: {
+                Query: {
+                    fields: {
+                        // Apollo Client keeps lists separate,
+                        // because they represent the results of queries
+                        // with different variable values.
+                        // We need Apollo Client to instead merge tickets from our fetchMore query
+                        // with the tickets from our original query
+                        // See more at https://www.apollographql.com/docs/tutorial/queries/
+                        allTicketChanges: concatPagination(),
```

Comment:
can we use it by default for all lists?

Reply:
No, because different lists have different pagination types.
Tickets has "window"- like pagination, when we a displaying a slice of data.
TicketChangea has "merge"-like pagination, when we are displaying a list, that grows by subsequent loading of more items.